lib/streamlit/DeltaGenerator.py

```diff
@@ -2255,16 +2259,15 @@ def _maybe_melt_data_for_add_rows(data, delta_type, last_index):
             data = data_frame_proto.convert_anything_to_df(data)
 
         if type(data.index) is pd.RangeIndex:
-            old_step = _get_pandas_index_attr(data, 'step')
+            old_step = _get_pandas_index_attr(data, "step")
 
             # We have to drop the predefined index
             data = data.reset_index(drop=True)
 
-            old_stop = _get_pandas_index_attr(data, 'stop')
+            old_stop = _get_pandas_index_attr(data, "stop")
 
             if old_step is None or old_stop is None:
-                raise AttributeError("'RangeIndex' object has no attribute "
-                                     "'step'")
+                raise AttributeError("'RangeIndex' object has no attribute " "'step'")
```

Comment:
The linter messed up here. If it fits within 80 characters, can you join these strings?

Reply:
doesn't fit :/ 
"""
                raise AttributeError("'RangeIndex' object has no attribute "
                                     "'step'")
'''